### ISSUE TITLE
Feature Request: Hitting Enter uses the highlighted 'current' property instead of text input

### ISSUE THREAD

**Issue by Tristan-Goucher** [schultzcole/FVTT-Autocomplete-Inline-Properties#15]
In a similar vein of workflow smoothness to my other issue (#14 ), I've noticed that even if you have a property highlighted in the auto-complete and hit Enter, it only included what you've typed into the search box. For example, if I'm typing 'attributes.hp.tot', then 'attributes.hp.total' is highlighted in the popup, but if I hit Enter now, only 'attributes.hp.tot' will be put into the text area. Is there a way to store the index of the highlighted property in the preview list and then grab the text at that spot when you press Enter?

**Comment by schultzcole** [comment]
Yeah, this could be better. I suppose if there aren't any more layers and the user presses Enter it would be reasonable to autocomplete the last part of the key
